broken-line/contracts/LibBrokenLine.sol

```diff
@@ -51,8 +65,52 @@ library LibBrokenLine {
     }
 
     /**
-     * Обновляет initial линию для переданного time. Высчитывает и применяет все изменения из slopeChanges за этот период
-    **/
+     *remove Line from BrokenLine, return line.bias, which actual now moment
+     **/
+    function remove(BrokenLineDomain.BrokenLine storage brokenLine, uint id, uint toTime) internal returns (uint) {
+        BrokenLineDomain.LineData memory lineData = brokenLine.initiatedLines[id];
+        BrokenLineDomain.Line memory line = lineData.line;
+        require(line.bias != 0, "Line with given id already finished");
+
+        update(brokenLine, toTime);
+        //check time Line is over
+        uint period = line.bias.div(line.slope);
+        uint finishTime = line.start.add(period).add(lineData.cliff);
+        if (toTime > finishTime) {
+            line.bias = 0;
+            return 0;
+        }
+        uint finishTimeMinusOne = finishTime.sub(1);
+        int mod = safeInt(line.bias.mod(line.slope));
+        uint nowBias = line.bias;
+        uint cliffEnd =  line.start.add(lineData.cliff).sub(1);
+        if (toTime <= cliffEnd) { //cliff works
+            //in cliff finish time compensate change slope by oldLine.slope
+            brokenLine.slopeChanges[cliffEnd] = brokenLine.slopeChanges[cliffEnd].sub(safeInt(line.slope));
+            //in new Line finish point use oldLine.slope
+            brokenLine.slopeChanges[finishTimeMinusOne] = brokenLine.slopeChanges[finishTimeMinusOne].add(safeInt(line.slope)).sub(mod);
+        } else { //cliff finish
```

Comment:
let's write it else if {} else {}. so all three cases will be be on the same level

Reply:
done by @evgenynacu

Reply:
agree